packages/ui-library/src/introduction.md

```diff
@@ -0,0 +1,33 @@
+# Welcome to the Yoast UI Library
+The Yoast UI library is a React component library for building Yoast user interfaces. This Storybook provides an interactive overview of all available components and examples on how to use them.
+
+[View the changelog here](https://github.com/Yoast/wordpress-seo/blob/trunk/packages/ui-library/changelog.md).
```

Comment:
Not sure if this is better. But since we are in the Storybook, and you included the changelog here too, you can:
```suggestion
[View the changelog here](/story/changelog--page).
```
However, the mix between these addons don't seem to work right. If you prefix with `/docs/` you get an error. And this `/story/` goes to the full page. But might be nicer anyway?